### DISCUSSION TITLE
App funktiniert nicht mehr?

### DISCUSSION THREAD

**Discussion by biker1602** [roleoroleo/yi-hack-Allwinner-v2#729] [Q&A]
Hallo
Ich habe mal eine Frage. Wenn ich unter Settings Disable Cloud und Recording without Cloud aktiviere, bekomme ich in der App nach kurzer Zeit keine Verbindung mehr zur Kamera. Sobald ich die beiden Punkte wieder deaktiviere, bekomme ich in der App auch wieder ein Bild. Ist das normal oder muss ich in den Einstellungen der App oder yi-hack Konfiguration etwas ändern? Ich möchte ja die App parallel ja trotzdem weiterhin nutzen.

**Comment by roleoroleo** [comment]
You can't use the app if you disable the cloud.
It's the normal behavior.
